### PR TITLE
Upgrade coursier-small to 1.3.3

### PR DESCRIPTION
**Pull request by sideeffffect**
hopefully fixes https://github.com/scalacenter/scalafix/issues/930

### CODE REVIEW

File: src/main/scala/scalafix/sbt/ScalafixPlugin.scala [hunk 1]
```diff
@@ -4,11 +4,8 @@ import java.nio.file.Path
 import java.{util => jutil}
 
 import com.geirsson.coursiersmall.Repository
-import java.io.PrintStream
-
 import sbt.Keys._
-import sbt.internal.sbtscalafix.Compat
-import sbt.internal.sbtscalafix.JLineAccess
+import sbt.internal.sbtscalafix.{Compat, JLineAccess}
```
Comment (olafurpg) [inline src/main/scala/scalafix/sbt/ScalafixPlugin.scala]: Please try to avoid unrelated diffs like this, it makes it easier to review

**Comment by olafurpg** [comment]
> hopefully fixes scalacenter/scalafix#930

Is there a reason you can't validate locally whether this change fixes the issue?

**Comment by sideeffffect** [comment]
@olafurpg it would probably be possible somehow, but it's always easier to test jars which are already published to Maven central -- especially in a corporate setting, things are always more complicated here
